From da22ca4346c0ebf6198e0ac34d9329fcd76aa935 Mon Sep 17 00:00:00 2001
From: jeremyd2019 <github@jdrake.com>
Date: Sat, 22 Jan 2022 11:33:00 -0800
Subject: [PATCH 068/N] mingw_smoketests: fix _UCRT condition

The prior detection missed ARM (32 and 64) being UCRT.  Since there are fewer non-UCRT platforms, list those two instead of trying to extend the list of UCRT.
---
 mingw_smoketests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mingw_smoketests.py b/mingw_smoketests.py
index d92c74a..ce95846 100644
--- a/mingw_smoketests.py
+++ b/mingw_smoketests.py
@@ -34,7 +34,7 @@ if "MSYSTEM" in os.environ:
 else:
     SEP = "\\"
 
-_UCRT = "clang" in sysconfig.get_platform() or "ucrt" in sysconfig.get_platform()
+_UCRT = sysconfig.get_platform() not in ('mingw_x86_64', 'mingw_i686')
 
 
 class Tests(unittest.TestCase):
